pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -312,7 +312,8 @@ def get_evaluate_epoch_results(self, test_mode):
 
         # clear mem
         self.eval_loop_results = []
-        return results
+
+        return self.trainer.predictions.attach_predictions(results, self._current_stage)
```

Comment:
can user skip this action, it seems to be executed always...

Reply:
If he didn't use `add_predictions`, this function won't do anything.

Reply:
so far I see, it will, just maybe it would run on an empty set, right?

Reply:
Yes, it will run on an empty set as no prediction were added.

Reply:
I think it would be clearer and easier for a user to have it skip instead of spending some time on search/debug what the func does and understand all the chancing when we do not use it...

Reply:
Ok.